README.md

```diff
@@ -60,6 +60,61 @@ It's a good idea to publish pre-built binaries of your module if you want others
 1. In the `package.json`, update the `"binary"` field to the appropriate s3 bucket `host`.
 1. If you plan on publishing binaries via Travis-CI, [setup your AWS security credentials in the `.travis.yml`](https://github.com/mapbox/node-pre-gyp#travis-automation) - this gives the travis environment the proper access to the bucket named above.
 
+For Mapbox staff we recommend publishing to `s3://mapbox-node-binary/<your module>`.
+
+1) Copy the ci.template.js
+
+Copy the `cloudformation/ci.template.js` into your repo.
+
+2) Create a user with permissions to upload to s3://mapbox-node-binary/<your module name>/
+
+First configure your AWS creds with `mbx auth`
+
+Then run:
+
+```bash
+npm install @mapbox/cfn-config
+npm install @mapbox/cloudfriend
+./node_modules/.bin/validate-template cloudformation/ci.template.js
+./node_modules/.bin/build-template cloudformation/ci.template.js > cloudformation/ci.template
+./node_modules/.bin/cfn-config create ci-binary-publish cloudformation/ci.template -c cfn-configs
+```
+
+3) Encode aws keys in .travis.yml
+
+First export your config with mbxcli (`mbx export-stack-config ci-binary-publish`).
```

Comment:
I've revised the docs to show how to do this with the AWS UI instead. Hopefully that will resolve this issue: https://github.com/mapbox/node-cpp-skel/pull/41/commits/ffe3deeb8f4227e40ad9a571a0a80f58b6535165. @TheMarex can you test again?